src/testdir/test_termdebug.vim

```diff
@@ -179,6 +179,60 @@ func Test_termdebug_basic()
   %bw!
 endfunc
 
+func Test_termdebug_decimal_breakpoints()
+  let bin_name = 'XTD_basic'
```

Comment:
Shouldn't a different `bin_name` be used?

Reply:
No, it sets function local variables in a function. I was suggesting changing `XTD_basic` to something else.

Reply:
> No, it sets function local variables in a function.

Auch! I am familiar with Vim9 but with the legacy language I have some problem, sorry!
 
Btw, I changed the name of the file.

Reply:
when you change it here, you should also change it a few lines below, in the `Termdebug ./bin_name`, no?

Reply:
Usch! Sorry. I also run the test locally and for some reason they passed... fixed now. Please, take a look. :)